app/src/main/java/org/jellyfin/androidtv/ui/SettingsPopup.kt

```diff
@@ -0,0 +1,72 @@
+package org.jellyfin.androidtv.ui
+
+import android.content.Context
+import android.graphics.Color
+import android.graphics.drawable.ColorDrawable
+import android.view.Gravity
+import android.view.LayoutInflater
+import android.view.View
+import android.widget.PopupWindow
+import android.widget.TextView
+import org.jellyfin.androidtv.databinding.SettingsPopupBinding
+import org.jellyfin.androidtv.util.Utils
+
+class SettingsPopup(
+	context: Context?,
```

Comment:
```suggestion
	context: Context,
```

So you don't have to do `context!!` everywhere